### PR TITLE
Calculator events

### PR DESCRIPTION
**Pull request by Ursaaaaaa**
- Copied the files calculate.js and operate.js into a logic/ directory in your project.
- Analyzed the files calculate.js and operate.js, be sure you understand what they do
- Imported the files in your Calculator component.
- Implemented the event handlers you need to use the math logic from calculate.js and operate.js in your React component.

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -1,36 +1,61 @@
 import React from 'react';
 import './calc.css';
+import calculate from '../logic/calculate';
```
Comment (mateo951) [inline src/components/calculator.js:3]: The Calculator component is imported from the calculate.js file as expected 💣

File: src/components/calculator.js [hunk 2]
```diff
@@ -1,36 +1,61 @@
...
+          <button onClick={this.handleClicks} type="button" className="btn" value="5">5</button>
+          <button onClick={this.handleClicks} type="button" className="btn" value="6">6</button>
+          <button onClick={this.handleClicks} type="button" className="btn btnaction" value="-">-</button>
+          <button onClick={this.handleClicks} type="button" className="btn" value="1">1</button>
+          <button onClick={this.handleClicks} type="button" className="btn" value="2">2</button>
+          <button onClick={this.handleClicks} type="button" className="btn" value="3">3</button>
+          <button onClick={this.handleClicks} type="button" className="btn btnaction" value="+">+</button>
+          <button onClick={this.handleClicks} type="button" className="btnZero" value="0">0</button>
+          <button onClick={this.handleClicks} type="button" className="btn" value=".">.</button>
+          <button onClick={this.handleClicks} type="button" className="btn btnaction" value="=">=</button>
```
Comment (mateo951) [inline src/components/calculator.js:58]: Well done 🔥 You implement all calculator operations with the click event 👌

**Review by mateo951 — APPROVED** [review]
### Hi @ramrod433, 

📝👌 **Your project highlights** 🔥😎 
- General Requirements ✔
- All operations are working 🙌
- You used events very well 🔥
- Good use of lifecycle methods 💪😎

Your project is complete!  There is nothing else to say other than... it's time to merge it 🤓 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.js (hunk 2 above)
Comment (Ursaaaaaa) [reply src/components/calculator.js:58]: Thank you

**Pull request merged**